plenum/cli/cli.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, looper, basedirpath, nodeReg=None, cliNodeReg=None,
         self.plugins = {}
         self.pluginPaths = []
         self.defaultClient = None
-        self.activeIdentifier = None
```

Comment:
Looks like not all `identifier` and `keyring` variable findings are replaced by `DID` and `wallet`. Is it intentional?